jobs/build_docker/prepare_docker_post_test.sh

```diff
@@ -61,3 +61,25 @@ set -e
 
 docker-compose -f docker-compose-mini.yml up > $WORKSPACE/build-log/vagrant.log &
 popd
+
+#"common" folder is the deepest folder in mount folder which is used to share files on docker
+#check the "common"folder is used to make sure all folder is created (all mount opreation is done),then change the authority
```

Comment:
other than "authority", I think more common saying is "permissions"  or "privileges"